diff --git a/Vagrantfile b/Vagrantfile
index e5d9e18..d79f9f6 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -3,12 +3,16 @@
 
 Vagrant.configure("2") do |config|
 
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 2048
+    v.cpus = 2
+  end
   config.vm.box = "precise64"
-  config.vm.box_url = "https://oss-binaries.phusionpassenger.com/vagrant/boxes/ubuntu-12.04.3-amd64-vmwarefusion.box"
+  config.vm.box_url = "https://oss-binaries.phusionpassenger.com/vagrant/boxes/latest/ubuntu-12.04-amd64-vbox.box"
 
   config.vm.hostname = "vagrantpress"
-  config.dns.tld = "wpdev"
-  config.dns.patterns = [/^.*\.wpdev$/]
+  #config.dns.tld = "wpdev"
+  #config.dns.patterns = [/^.*\.wpdev$/]
 
   config.vm.network :private_network, ip: "172.16.23.16"
 
@@ -24,5 +28,5 @@ Vagrant.configure("2") do |config|
   #   https://github.com/mitchellh/vagrant/issues/2756
   #
   # Hopefully this can be removed once Vagrant >= 1.4.4 is out.
-  VagrantDNS::Config.auto_run = false
+  #VagrantDNS::Config.auto_run = false
 end
